<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mottar</class>
 <widget class="QWidget" name="mottar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="side_motta">
      <widget class="QPushButton" name="tilbake_0_motta">
       <property name="geometry">
        <rect>
         <x>1130</x>
         <y>670</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>341</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="modusogbehandling">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Valgt modus: Motta</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>23</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Velg overflatebehandling</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="x_0_motta">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="herding">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>420</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Herding</string>
       </property>
      </widget>
      <widget class="QPushButton" name="lakkering">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>160</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Lakkering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="elektrokjemisk_metode">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>160</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Elektrokjemisk metode</string>
       </property>
      </widget>
      <widget class="QPushButton" name="kjemisk_metode">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>160</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Kjemisk metode</string>
       </property>
      </widget>
      <widget class="QPushButton" name="annet">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>420</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Annet</string>
       </property>
      </widget>
      <widget class="QLabel" name="logo_tepvr_0">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="side_lakkering_motta">
      <widget class="QLabel" name="modus_1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Valgt modus: Motta | Lakkering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tilbake_1_lakk_motta">
       <property name="geometry">
        <rect>
         <x>1130</x>
         <y>670</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pulverlakkering">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>210</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pulverlakkering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="vaatlakkering">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>210</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Våtlakkering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="offshore_lakkering">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>210</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Offshore lakkering</string>
       </property>
      </widget>
      <widget class="QLabel" name="logo_tepvr_1">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="side_elektrokjemisk_metode_motta">
      <widget class="QLabel" name="modus_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>0</y>
         <width>411</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Valgt modus: Motta | Elektrokjemisk Metode</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tilbake_2_elkj_motta">
       <property name="geometry">
        <rect>
         <x>1130</x>
         <y>670</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="hardforkromming">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>130</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Hardforkromming</string>
       </property>
      </widget>
      <widget class="QPushButton" name="elzink">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>390</y>
         <width>661</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Elektrolytisk forsinking/elzink</string>
       </property>
      </widget>
      <widget class="QPushButton" name="eloksering_alu">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Eloksering (aluminium)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="kromatering">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>130</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Kromatering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="elektrolytisk_fornikling">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>390</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Elektrolytisk fornikling</string>
       </property>
      </widget>
      <widget class="QLabel" name="logo_tepvr_2">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="side_kjemisk_metode_motta">
      <widget class="QLabel" name="modus_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>341</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Valgt modus: Motta | Kjemisk Metode</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tilbake_3_kj_motta">
       <property name="geometry">
        <rect>
         <x>1130</x>
         <y>670</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="syrevask">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>150</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Syrevask</string>
       </property>
      </widget>
      <widget class="QPushButton" name="brunering">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>150</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Brunering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="galvanisering_varmforsinking">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>410</y>
         <width>661</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Galvanisering/varmforsinking</string>
       </property>
      </widget>
      <widget class="QPushButton" name="fosfatering">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Fosfatering</string>
       </property>
      </widget>
      <widget class="QLabel" name="logo_tepvr_3">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="side_herding_motta">
      <widget class="QLabel" name="modus_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>271</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Valgt modus: Motta | Herding</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tilbake_4_herding_motta">
       <property name="geometry">
        <rect>
         <x>1130</x>
         <y>670</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="herd_herding">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>180</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Herding</string>
       </property>
      </widget>
      <widget class="QPushButton" name="nitrering">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>180</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nitrering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sett_herding">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>180</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sett-herding</string>
       </property>
      </widget>
      <widget class="QPushButton" name="plasma_nitrering">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>440</y>
         <width>311</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Plasma nitering</string>
       </property>
      </widget>
      <widget class="QLabel" name="logo_tepvr_4">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="side_annet_motta">
      <widget class="QLabel" name="modus_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>0</y>
         <width>261</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Valgt modus: Motta | Annet</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tilbake_5_annet_motta">
       <property name="geometry">
        <rect>
         <x>1130</x>
         <y>670</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sandblaasing">
       <property name="geometry">
        <rect>
         <x>930</x>
         <y>100</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sandblåsing</string>
       </property>
      </widget>
      <widget class="QPushButton" name="traadskjaering">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>100</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Trådskjæring</string>
       </property>
      </widget>
      <widget class="QPushButton" name="polering">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>500</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Polering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="gullbelegging">
       <property name="geometry">
        <rect>
         <x>930</x>
         <y>300</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Gullbelegging</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sliping">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sliping</string>
       </property>
      </widget>
      <widget class="QPushButton" name="glass_blaasing">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>100</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Glass blåsing</string>
       </property>
      </widget>
      <widget class="QPushButton" name="fortinning">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>300</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Fortinning</string>
       </property>
      </widget>
      <widget class="QPushButton" name="elektrolytisk_polering">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>500</y>
         <width>391</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Elektrolytisk polering</string>
       </property>
      </widget>
      <widget class="QPushButton" name="forsolving_gullbelegging">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>300</y>
         <width>391</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Forsølving/Gullbelegging</string>
       </property>
      </widget>
      <widget class="QLabel" name="logo_tepvr_5">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>0</y>
         <width>481</width>
         <height>94</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
